src/OpenDiffix.Core/Aggregator.fs

```diff
@@ -288,6 +288,115 @@ type private DiffixLowCount() =
       else
         Boolean(Anonymizer.isLowCount aggContext.AnonymizationParams anonContext state)
 
+type private DiffixSum() =
+  let nullState: Anonymizer.SumState = { Positive = null; Negative = null; IsReal = false }
+  let mutable state = nullState
+
+  let initialState length : Anonymizer.SumState =
+    { nullState with
+        Positive =
+          Array.init length (fun _ -> { AidContributions = Dictionary<AidHash, float>(); UnaccountedFor = 0.0 })
+        Negative =
+          Array.init length (fun _ -> { AidContributions = Dictionary<AidHash, float>(); UnaccountedFor = 0.0 })
+    }
+
+  let increaseUnaccountedFor valueIncrease i =
+    let absValueIncrease = abs (valueIncrease)
+
+    if valueIncrease > 0.0 then
+      state.Positive.[i].UnaccountedFor <- state.Positive.[i].UnaccountedFor + absValueIncrease
+    else if valueIncrease < 0.0 then
+      state.Negative.[i].UnaccountedFor <- state.Negative.[i].UnaccountedFor + absValueIncrease
+    else
+      state.Positive.[i].UnaccountedFor <- state.Positive.[i].UnaccountedFor + absValueIncrease
+      state.Negative.[i].UnaccountedFor <- state.Negative.[i].UnaccountedFor + absValueIncrease
```

Comment:
Isn't `absValueIncrease` zero here? Why add it?

Reply:
I simplified this altogether and forgot to push the change before review :facepalm:. I'll fix with the other fixes